tests/PHPUnit/Integration/ReleaseCheckListTest.php

```diff
@@ -617,6 +616,47 @@ public function test_TotalPiwikFilesSize_isWithinReasonnableSize()
         $this->assertGreaterThan($minimumTotalFilesizesExpectedInMb * 1024 * 1024, $sumFilesizes, "expected to have at least $minimumTotalFilesizesExpectedInMb Mb of files in Piwik codebase.");
     }
 
+    public function test_noUpdatesInCorePlugins()
+    {
+        $manager = Manager::getInstance();
+        $plugins = $manager->loadAllPluginsAndGetTheirInfo();
+
+        $pluginsWithUnexpectedUpdates = array();
+        $pluginsWithUpdates = array();
+        $numTestedCorePlugins = 0;
+        foreach ($plugins as $pluginName => $info) {
+            if ($manager->isPluginBundledWithCore($pluginName)) {
```

Comment:
Wondering if that might start failing if we put updates into TagManager. I guess we still want to have them separate in that case, right?

Reply:
Right, changed that 👍